cmake/common.cmake

```diff
@@ -40,6 +40,7 @@ macro(irr_create_executable_project _EXTRA_SOURCES _EXTRA_OPTIONS)
 			PROPERTIES
 			RUNTIME_OUTPUT_DIRECTORY_DEBUG "${PROJECT_SOURCE_DIR}/bin"
 			RUNTIME_OUTPUT_DIRECTORY_RELEASE "${PROJECT_SOURCE_DIR}/bin"
+			VS_DEBUGGER_WORKING_DIRECTORY "${PROJECT_SOURCE_DIR}/bin" # seems like has no effect
```

Comment:
if you dlete cmake cache and reconfigure+regenerate?

Reply:
tried, nothing anyway